src/services/auth/plugins/password/password.controller.test.ts

```diff
@@ -94,8 +94,14 @@ describe('Password', () => {
           captcha: MOCK_CAPTCHA,
         },
       });
-      expect(response.statusCode).toEqual(StatusCodes.SEE_OTHER);
-      expect(response.json()).toHaveProperty('resource');
+      expect(response.statusCode).toEqual(StatusCodes.NO_CONTENT);
+      expect(response.headers['set-cookie']).toBeDefined();
+
+      // last authenticated at should be updated
+      const m = await db.query.accountsTable.findFirst({
+        where: eq(accountsTable.email, actor.email!),
+      });
+      expect(m?.lastAuthenticatedAt).not.toEqual(actor.lastAuthenticatedAt);
```

Comment:
Should we ensure the new `last_authenticated_at` is `after` the old one ? I think we can use functions from `date-fns` for this.